src/server.ts

```diff
@@ -42,15 +44,16 @@ class Server {
   }
 
   setupRoutes() {
-    this.app.use('/', router);
-    this.app.use('/',routes.protectedRoutes);
-    this.app.use('/',routes.publicRoutes);
+    const route = new Routes();
+    this.app.use("/", router);
+    this.app.use("/", route.publicRoutes());
+    this.app.use("/", route.protectedRoutes());
```

Comment:
there is no requirement to create an object of a class that does not have any state (which means does not have any variables)